src/SprykerSdk/Acp/Schema/SchemaConverterInterface.php

```diff
@@ -5,15 +5,16 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace SprykerSdk\Acp\Configuration\SchemaGenerator;
+namespace SprykerSdk\Acp\Schema;
 
-interface SchemaGeneratorInterface
+use cebe\openapi\spec\OpenApi;
+
+interface SchemaConverterInterface
```

Comment:
Would `SchemaReader`or `SchemaParser` be a more appropriate name, since we are "converting" to an internal format?